### ISSUE TITLE
Pagination should be documented

### ISSUE THREAD

**Issue by JakeAustwick** [rubysherpas/forem#148]
Forem supports pagination using either Kamari or will_paginate, however this is not documented anywhere.

**Comment by parndt** [comment]
Thanks.. Any interest in documenting it? :)

**Comment by JakeAustwick** [comment]
I'll try document this later this week.

**Comment by jgadbois** [comment]
Will this actually work with will_paginate?  A couple controllers call the `per` method which I believe is Kaminari only

**Comment by jgadbois** [comment]
Never mind, just saw the will_paginate monkey patch

**Comment by parndt** [comment]
@JakeAustwick any luck?

**Comment by JakeAustwick** [comment]
Not sure what the above commits are about, don't seem relevant to this issue.

I also just edited the wiki directly. I've never edited documentation before and presumed it would have created something similar to a pull request, for the wiki page. It seems it went straight through.

Anyway, There wasn't really must to write about so I just added a small section to the bottom of [this page](https://github.com/radar/forem/wiki/Configuration), feel free to improve / edit as necessary.
